AK/Array.h

```diff
@@ -103,21 +103,25 @@ struct Array {
     {
         static_assert(Size > 0, "No values to max() over");
 
-        T value = __data[0];
+        size_t candidate = 0;
         for (size_t i = 1; i < Size; ++i)
```

Comment:
Can you put curly braces around both `for` body?

Reply:
I can, but I got the impression it's not like that in the codebase.

Reply:
https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#omission-of-curly-braces-from-statement-blocks